packages/contracts/src/dump.ts

```diff
@@ -0,0 +1,131 @@
+/* External Imports */
+import * as path from 'path'
+import { ethers } from 'ethers'
+import * as Ganache from 'ganache-core'
+import { deployAllContracts, RollupDeployConfig } from './deployment'
+
+interface StorageDump {
+  [key: string]: string
+}
+
+export interface StateDump {
+  contracts: {
+    ovmExecutionManager: string
+    ovmStateManager: string
+  }
+  accounts: {
+    [address: string]: {
+      balance: number
+      nonce: number
+      code: string
+      storage: StorageDump
+    }
+  }
+}
+
+const getChangedAccounts = async (cStateManager: any): Promise<string[]> => {
+  return new Promise<string[]>((resolve, reject) => {
+    const accounts: string[] = []
+    const stream = cStateManager._trie.createReadStream()
+
+    stream.on('data', (val: any) => {
+      accounts.push(val.key.toString('hex'))
+    })
+
+    stream.on('end', () => {
+      resolve(accounts)
+    })
+  })
+}
+
+const getStorageDump = async (
+  cStateManager: any,
+  address: string
+): Promise<StorageDump> => {
+  return new Promise<StorageDump>((resolve, reject) => {
+    cStateManager._getStorageTrie(address, (err: any, trie: any) => {
+      if (err) {
+        reject(err)
+      }
+
+      const storage: StorageDump = {}
+      const stream = trie.createReadStream()
+
+      stream.on('data', (val: any) => {
+        storage[val.key.toString('hex')] = val.value.toString('hex')
+      })
+
+      stream.on('end', () => {
+        resolve(storage)
+      })
+    })
+  })
+}
+
+export const makeStateDump = async (): Promise<any> => {
+  const ganache = Ganache.provider({
+    gasLimit: 100_000_000,
+    allowUnlimitedContractSize: true,
+    accounts: [
+      {
+        secretKey:
+          '0x29f3edee0ad3abf8e2699402e0e28cd6492c9be7eaab00d732a791c33552f797',
+        balance: 10000000000000000000000000000000000,
+      },
+    ],
+  })
+
+  const provider = new ethers.providers.Web3Provider(ganache)
+  const signer = provider.getSigner(0)
+
+  const config: RollupDeployConfig = {
+    signer,
+    rollupOptions: {
+      forceInclusionPeriodSeconds: 600,
+      ownerAddress: await signer.getAddress(),
+      sequencerAddress: await signer.getAddress(),
+      gasMeterConfig: {
+        ovmTxFlatGasFee: 0,
+        ovmTxMaxGas: 1_000_000_000,
+        gasRateLimitEpochLength: 600,
+        maxSequencedGasPerEpoch: 1_000_000_000_000,
+        maxQueuedGasPerEpoch: 1_000_000_000_000,
+      },
+      deployerWhitelistOwnerAddress: await signer.getAddress(),
+      allowArbitraryContractDeployment: true,
```

Comment:
I feel like this option was removed from being useful since `initialize(address,bool)` needs to be called on the contract before the contract does anything, unless this will send an additional transaction initializing.

Reply:
Yeah I'm not actually sure if it's even used in the deployment, but it's currently a required deployment configuration option. Do you think better to update the deploy config interface? Or just keep the option here for now and do that in another PR?